driver/src/devicedrv/mali/linux/mali_osk_time.c

```diff
@@ -46,12 +46,10 @@ void _mali_osk_time_ubusydelay(u32 usecs)
 
 u64 _mali_osk_time_get_ns(void)
 {
-	struct timespec tsval;
-	getnstimeofday(&tsval);
-	return (u64)timespec_to_ns(&tsval);
+	return timespec64_to_ns;
```

Comment:
Was this even build tested? you need `()` at the end, otherwise you would return function pointer. Same below.